runtime/cfdumper/main.c

```diff
@@ -1437,12 +1437,22 @@ static void printDisassembledMethod(J9CfrClassFile* classfile, J9CfrMethod* meth
 			NEXT_U8_ENDIAN(bigEndian, bc, bcIndex);
 			if(wide)
 			{
-				j9tty_printf( PORTLIB, "%s ", sunJavaBCNames[bc]);
+				j9tty_printf( PORTLIB, "%s", sunJavaBCNames[bc]);
 			}
 			else
 			{
-				j9tty_printf( PORTLIB, "%5i %s ", pc, sunJavaBCNames[bc]);
+				j9tty_printf( PORTLIB, "%5i %s", pc, sunJavaBCNames[bc]);
```

Comment:
Same here.

Reply:
Same reason as above: For including bytecode number in output